ansible/roles/vm_set/tasks/main.yml

```diff
@@ -98,18 +98,18 @@
         - libvirt-daemon-system
         - qemu-system-x86
       become: yes
-      when: host_distribution_version.stdout == "20.04"
+      when: host_distribution_version.stdout == "20.04" or host_distribution_version.stdout == "22.04"
   when: package_installation|bool
 
 - name: Get default pip_executable
   set_fact:
     pip_executable: pip
-  when: pip_executable is not defined and host_distribution_version.stdout != "20.04"
+  when: pip_executable is not defined and host_distribution_version.stdout != "20.04" and host_distribution_version.stdout != "22.04"
 
 - name: Get default pip_executable
   set_fact:
     pip_executable: pip3
-  when: pip_executable is not defined and host_distribution_version.stdout == "20.04"
+  when: pip_executable is not defined and host_distribution_version.stdout == "20.04" or host_distribution_version.stdout == "22.04"
```

Comment:
Should the condition be like this:
```
when: pip_executable is not defined and (host_distribution_version.stdout == "20.04" or host_distribution_version.stdout == "22.04")
```

Reply:
> Should the condition be like this:
> 
> ```
> when: pip_executable is not defined and (host_distribution_version.stdout == "20.04" or host_distribution_version.stdout == "22.04")
> ```

@wangxin Thanks for pointing it out. Done.